build/kured/Dockerfile

```diff
@@ -14,16 +14,13 @@ WORKDIR /go/src/github.com/weaveworks/kured
 RUN export GOOS=$(echo ${TARGETPLATFORM} | cut -d / -f1) && \
     export GOARCH=$(echo ${TARGETPLATFORM} | cut -d / -f2) && \
     export GOARM=$(echo ${TARGETPLATFORM} | cut -d / -f3 | cut -c2-) && \
-    curl -L -o /usr/bin/kubectl "https://storage.googleapis.com/kubernetes-release/release/${KUBECTL_VERSION}/bin/linux/${GOARCH}/kubectl" && \
     git clone --depth 1 -b ${VERSION} https://github.com/weaveworks/kured.git . && \
-    go build -ldflags "-X main.version=${VERSION}" -o kured ./cmd/kured
+    go build -ldflags "-X main.version=${VERSION}" -o kured ./cmd/kured/*.go && \
+    go test ./...
 
-FROM alpine:3.12.1
+FROM alpine:3.13
 
-RUN apk add --no-cache tzdata
-
-COPY --from=build /usr/bin/kubectl /usr/bin/kubectl
```

Comment:
Is this no longer required?

Reply:
Kubectl is indeed no longer required.

Reply:
https://github.com/weaveworks/kured/commit/72c4112e2064bec2096ab450c3fb13876d7f3b68